db/migrations/011.do.update-indexes.sql

```diff
@@ -0,0 +1,10 @@
+-- Step 1: Drop old indexes referencing participant_address
+DROP INDEX IF EXISTS daily_reward_transfers_to_address_day;
+DROP INDEX IF EXISTS idx_daily_scheduled_rewards_participant_address;
+
+-- Step 2: Create new indexes for better performance (AFTER dropping old ones)
+CREATE INDEX CONCURRENTLY idx_daily_scheduled_rewards_pid_day 
```

Comment:
When I run `npm migrate` on my computer, I get the following error:

```
error: CREATE INDEX CONCURRENTLY cannot run inside a transaction block
```

I am afraid we need to split this schema migration file into three files: 1) drop old indices 2) create first index 3) create second index.

I'll apply the change myself to speed up landing of this PR.